app/views/moderations/actions_panel.html.erb

```diff
@@ -190,30 +190,43 @@
       </a>
     </div>
 
-    <% if policy(@moderatable).can_perform_moderator_actions? %>
-      <button aria-haspopup="true" aria-expanded="false" aria-controls="admin-action-options" class="other-things-btn admin-action" type="button" data-other-things-type="admin-action" aria-label="<%= t("views.moderations.actions.admin.aria_label") %>">
-        <div class="label-wrapper">
-          <div class="icon circle centered-icon">
-            <%= inline_svg_tag("admin.svg", aria_hidden: true, title: t("views.moderations.actions.admin.icon")) %>
-          </div>
-          <header>
-            <h2><%= t("views.moderations.actions.admin.heading") %></h2>
-            <h3><%= t("views.moderations.actions.admin.subtitle", user: user_username) %></h3>
-          </header>
-        </div>
-        <div class="toggle-chevron-container">
-          <%= inline_svg_tag("arrow-down-fill.svg", aria: true, title: t("views.moderations.actions.toggle")) %>
+    <button aria-haspopup="true" aria-expanded="false" aria-controls="admin-action-options" class="other-things-btn admin-action" type="button" data-other-things-type="admin-action" aria-label="<%= t("views.moderations.actions.admin.aria_label") %>">
+      <div class="label-wrapper">
+        <div class="icon circle centered-icon">
```

Comment:
According to my truncated-project proposal [discussed here](https://forem-team.slack.com/archives/C03CHBBMTJP/p1658500957993839), this entire section (Moderator actions) would be hidden from the Mod Panel when the user is in the Mod Center.

The 3 action in this section are not working in the Mod Center, because these actions use a Preact modal for which the content is specific to the article, and is hidden in the individual article’s page. In the Mod Center, we have several articles, and this hidden data is not present, hence the modal fails to load.